todo/material/task.md

```diff
@@ -0,0 +1,110 @@
+# Aufgabenliste
+
+::: info
+Diese Aufgabe basiert auf _keinem CS50 Practice Problem_.
+
+**Disclaimer:** Diese Aufgabe wurde nicht vom Lehrstuhl herausgegeben und kann Fehler enthalten. Sie dient, wie das gesamte Material von [inf-lab.dev](https://inf-lab.dev), lediglich zu Übungszwecken!
+:::
+
+Wir alle haben verschiedene Aufgaben, jedoch ist es nicht immer einfach sich alle seine Aufgaben zu merken. Daher wollen wir in dieser Aufgabe eine ToDo-Liste entwickeln, welche uns erlaubt unsere Aufgaben über die Konsole zu verwalten.
+
+## Teilaufgabe 1
+
+Wie immer, wenn wir einen vorbereiteten Code bekommen, sollten wir diesen vorerst verstehen bevor wir Änderungen vornehmen können. Lies deshalb den Code in der Datei `todo.py` und beantworte insbesondere folgende Fragen. Wenn du noch etwas unsicher mit der Python Syntax bist, kannst du die vorgegebene Datei `todo.c` zur hilfe nehmen. Diese enthält den selben Code - nur für die Programmiersprache C.
```

Comment:
Wie immer, wenn wir einen vorbereiteten Code bekommen, sollten wir diesen vorerst verstehen bevor wir Änderungen vornehmen können. Das Material kann mit folgendem Befehl heruntergeladen werden:

```bash
wget <path to the material zip>
```

Lies den Code in der Datei `todo.py` und beantworte ...